src/lib/layer-manager.js

```diff
@@ -138,6 +167,42 @@ export default class LayerManager {
     return this;
   }
 
+  addEventListeners(eventNames) {
+    if (!this.eventManager) {
+      // eslint-disable-next-line max-len
+      log(1, 'The `canvas` element must be passed to the LayerManager constructor in order to add event listeners.');
+      return;
+    }
+
+    const eventHandlerMap = {
+      click: this._onClick,
+      pointermove: this._onPointerMove
+    };
+
+    const events = {};
+    eventNames.forEach(name => {
+      if (eventHandlerMap[name]) {
+        events[name] = eventHandlerMap[name];
+      } else {
+        log(1, `Event "${name}" is not supported.`);
+      }
+    });
+    this.eventManager.on(events);
+  }
+
+  removeEventListeners(element, eventNames) {
+    // TODO
+  }
+
+  validateEventHandling() {
```

Comment:
Yes -- after I wrote this I realized I needed to store `onLayerClick` and `onLayerHover` somewhere that `LayerManager` could get at them. That means now that all this logic can live within `LayerManager` instead of straddling across `deckgl.js` as well.